.github/workflows/e2e.yml

```diff
@@ -899,6 +900,7 @@ jobs:
           minimum-vertica-image: '24.3.0'
           use-cert: true
           cert-suffix: cert
+          use-cache: true
```

Comment:
They should not all being using the cache. We want some legs (1/2 of them) to run without cache.